Tensile/Common.py

```diff
@@ -2338,6 +2342,12 @@ def assignGlobalParameters( config ):
       tPrint(3, " %24s: %8s (unspecified)" % (key, defaultValue))
 
   globalParameters["ROCmPath"] = "/opt/rocm"
+  # /opt/rocm is not always the default location
+  if not os.path.isdir(globalParameters["ROCmPath"]):
+      if platform.system() == "Linux":
+          if distro.id() == "fedora":
+              # Fedora packages ROCm as part of its distro to system paths
+              globalParameters["ROCmPath"] = "/usr"
```

Comment:
The comment is also true of Debian, Gentoo and Ubuntu. IMO, `/usr` is a standard path that should be considered on any Linux, as otherwise, it sends us down a rabbit hole of adding distro names one-by-one (e.g., 'linuxmint' is effectively just Ubuntu for this purpose).

Reply:
ok i will remove the distro check and go with '/usr' for everyone linux

Reply:
you could also pass the prefix from cmake too (CMAKE_INSTALL_PREFIX)

Reply:
@Mystro256, it's fairly common to run Tensile from the source tree without building/installing it.